cypress/integration/components/Select.spec.js

```diff
@@ -136,6 +136,39 @@ describe("Select", () => {
     });
   });
 
+  describe.only("Multiselect: options live update", () => {
+    const selectTextInputSelector = "[data-testid='select-input'] input";
+    const getSelectTextInput = () => cy.get(selectTextInputSelector);
+
+    beforeEach(() => {
+      cy.renderFromStorybook("select--add-new-option-on-input-change");
+    });
+
+    it("add new option on every input change and input should be focused", () => {
+      getSelectTextInput().type("test");
+      getSelectTextInput().should("have.focus");
+
+      const csvValues = "PCN2, PCN1";
+
+      cy.paste({ destinationSelector: selectTextInputSelector, pastePayload: csvValues });
+
+      cy.focused().type("{backspace}{backspace}{backspace}");
+      getSelectTextInput().should("have.focus");
+
+      cy.focused().type("{enter}");
+      cy.focused().type("{enter}");
+    });
+
+    it("user able to paste and select values with live update of options", () => {
```

Comment:
This test seems very similar to the one above, where we type and assert focus. Is there anything else that it is testing?

Reply:
Yes, good point.
I've rethought this case and left only one test case.